101-bundle-json.md

```diff
@@ -442,22 +442,24 @@ Implementations MAY support user-defined additional actions as well. Such action
 
 ```json
 "actions": {
-    "status":{
+    "io.cnab.status":{
         "modifies": false,
         "description": "retrieves the status of an installation"
     },
-    "migrate":{
+    "io.cnab.migrate":{
         "modifies": false
     },
-    "dry-run":{
+    "io.cnab.dry-run":{
         "modifies": false,
         "stateless": true,
         "description": "prints what install would do with the given parameters values"
     }
 }
 ```
 
-The above declares to actions: `status` and `migrate`. This means that the associated invocation images can handle requests for `status` and `migrate` in addition to `install`, `upgrade`, and `uninstall`.
+The action _name_ SHOULD be namespaced and SHOULD use reverse domains notation - e.g. `com.example.myKey`. 
```

Comment:
Slight incongruity with the examples using hyphen-separators and the e.g. here using compoundWords. The spec is neutral about which to use, but [RFC4343 ](https://tools.ietf.org/html/rfc4343 ) states _Domain Name System (DNS) names are "case insensitive"_.

Reply:
Renamed to `com.example.action`